### PR TITLE
CANS-278: Only show the children in user's county

### PR DESCRIPTION
**Pull request by edgibbs**
This implements the functionality to redirect users and show warnings when they switch a county for a child.

https://osi-cwds.atlassian.net/browse/CANS-278

- [x] I used TDD to develop the feature
- [x] I have included unit tests
- [ ] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

- [x] My code follows the code style of this project.
- [x] I ran all my tests locally which were green.
- [x] My code adds no new issues to Code Climate
- [x] I ran all the linters for the project.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build (or bring donuts if I leave things broken), to check linters, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: app/javascript/Application/components/Client/ClientAddEditForm.test.js [hunk 1]
```diff
@@ -293,11 +345,11 @@ describe('<ClientAddEditForm />', () => {
 
   it('renders counties drop down', () => {
     const wrapper = getWrapperEdit();
-    const clientEditWrapper = wrapper.find('ClientAddEditForm').dive();
-    const childFormInstance = clientEditWrapper.instance();
-    childFormInstance.fetchCounties([{ id: '9', name: 'Fresno' }]);
-    clientEditWrapper.update();
-    const menuItems = clientEditWrapper.find('MenuItem');
+    const ClientEditWrapper = wrapper.find('ClientAddEditForm').dive();
```
Comment (denys-davydov) [inline app/javascript/Application/components/Client/ClientAddEditForm.test.js]: We are using camel case as a variables naming convention

Comment (edgibbs) [reply app/javascript/Application/components/Client/ClientAddEditForm.test.js]: good catch

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
Working on the failing build, looks related to chromedriver.

**Comment by edgibbs** [comment]
retest this please

**Pull request merged**